volunteer-app/www/templates/edit-log.html

```diff
@@ -41,6 +41,15 @@
             </label>
             <!-- /minutes -->
 
+            <!-- Project -->
+            <span class="input-label" ng-if="hasProjects">Project (optional)</span>
```

Comment:
There's an issue with how we are handling deleted Projects.

Testing Steps:
- add a project to a log
- see project appended to log display
- delete project
- project is no longer appended to log

This is not expected behaviour. Projects should be soft deleted

Reply:
What do you mean by "soft deleted"? When projects are deleted, `deleted_at` is set to the current date, and the foreign key linking a log to a project is set to null.

Reply:
From a UX point of view when you delete a project the only change you should see is that you cannot add this project to new logs.You should still be able to see a soft deleted project linked to old logs. It is part of historical data.

Reply:
OK sure, that can be done, but it doesn't have anything to do with the issues this PR is addressing (also I thought this whole deleted projects thing was an ongoing conversation).